hazelcast-client/src/test/java/com/hazelcast/client/MembershipListenerTest.java

```diff
@@ -123,16 +123,16 @@ public void addNullListener_thenException() throws Exception {
     }
 
 
-    private class MemberShipEventLoger implements MembershipListener {
+    private class MemberShipEventLogger implements MembershipListener {
 
-        private List<MembershipEvent> events = new Vector<MembershipEvent>();
+        public LinkedBlockingDeque<MembershipEvent> events = new LinkedBlockingDeque<MembershipEvent>();
```

Comment:
I'm fine with using a queue here, but you can also get the last element of a vector using vector.lastElement().